scripts/out/sunday.html

```diff
@@ -841,68 +2776,98 @@ <h2>Devrooms - Sunday</h2>
                         <p><a href="https://www.fosdem.org/2024/schedule/track/identity-and-access-management/" target="_blank" rel="noopener noreferrer">Schedule</a>
                         </p>
                     </div>
-                    <div class="btn"><a href="/#/room/#2024-k.3.401:fosdem.org">
-                        <div class="dark-panel button">Join</div>
+                    <div class="btn"><a href="/#/room/#2025-ua2.118_henriot_:fosdem.org">
+                        <div class="dark-panel button">Join Auditorium</div>
                     </a>
                     </div>
                 </div>
             
                 <div class="card">
                     <div class="room_text">
                         <!--<img src="https://via.placeholder.com/32">-->
-                        <p>Kernel devroom</p>
+                        <p>Embedded, Mobile and Automotive devroom</p>
                     </div>
                     <div class="room_subtext">
-                        <p><a href="https://www.fosdem.org/2024/schedule/track/kernel/" target="_blank" rel="noopener noreferrer">Schedule</a>
+                        <p><a href="https://www.fosdem.org/2024/schedule/track/embedded-mobile-and-automotive/" target="_blank" rel="noopener noreferrer">Schedule</a>
                         </p>
                     </div>
-                    <div class="btn"><a href="/#/room/#2024-ub5.132:fosdem.org">
-                        <div class="dark-panel button">Join</div>
+                    <div class="btn"><a href="/#/room/#2025-ub4.136:fosdem.org">
+                        <div class="dark-panel button">Join Auditorium</div>
                     </a>
                     </div>
                 </div>
             
                 <div class="card">
                     <div class="room_text">
                         <!--<img src="https://via.placeholder.com/32">-->
-                        <p>LLVM devroom</p>
+                        <p>APIs & friends devroom</p>
                     </div>
                     <div class="room_subtext">
-                        <p><a href="https://www.fosdem.org/2024/schedule/track/llvm/" target="_blank" rel="noopener noreferrer">Schedule</a>
+                        <p><a href="https://www.fosdem.org/2024/schedule/track/api/" target="_blank" rel="noopener noreferrer">Schedule</a>
                         </p>
                     </div>
-                    <div class="btn"><a href="/#/room/#2024-k.4.201:fosdem.org">
-                        <div class="dark-panel button">Join</div>
+                    <div class="btn"><a href="/#/room/#2025-k.3.201:fosdem.org">
```

Comment:
these links seem incorrect. They should go to track based names, not room based names.